core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -58,6 +71,113 @@
         left: 42px;
         position: absolute;
       }
+      @media only screen and (max-width : 600px)
+      {
+        .oppia-collection-player-title-font
+        {
+          font-size: 32px;
+        }
+      }
+      exploration-summary-tile md-card {
```

Comment:
Yes, because I want to remove the box-shadow only for the (current) preview card and not the blank preview card ie. blank card should have box-shadow.